libbeat/common/seccomp/policy_linux_amd64.go

```diff
@@ -34,6 +34,7 @@ func init() {
 					"arch_prctl",
 					"bind",
 					"brk",
+					"capget",
```

Comment:
@andrewkroh I didn't manage to update these files with `make seccomp`, it created new include files under metricbeat.
I have added this by hand here.